types/Config.js

```diff
@@ -117,6 +124,10 @@ export type Config = {|
 export type InitialConfig = {|
   automock?: boolean,
   bail?: boolean,
+  benchEnvironment?: string,
+  benchMatch?: Array<Glob>,
+  benchRegex?: string,
+  benchRunner?: string,
```

Comment:
I'd properly make an option "benchmark" available and within that, have `environment, match, runner`. I don't think we need to add the regex option for this new feature (we are only keeping `testRegex` for backwards compat).